services/thermal_mgr/thermal_mgr.c

```diff
@@ -42,20 +43,46 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 }
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
-  /* Send an event to the thermal manager queue */
-
+  // Send an event to the thermal manager queue
```

Comment:
check if the thermal mgr queue exists. if it doesn't, return err code invalid state